README.md

```diff
@@ -1,2 +1,62 @@
-# backend-interview-task
-Code for the backend interview task
+# Moneyhub Tech Test - Investments and Holdings
+
+At Moneyhub we use microservices to partition and separate the concerns of the codebase. In this exercise we have given you an example `admin` service and some accompanying services to work with. In this case the admin service backs a front end admin tool allowing non-technical staff to interact with data.
+
+A request for a new admin feature has been received
+
+## Requirements
+
+- An admin is able to generate a csv formatted report showing the values of all user holdings
+    - The report should be sent to the `/export` route of the holdings service
+    - The holdings service expects the report to be sent as csv text
+    - The csv should contain a row for each holding matching the following headers
+    |User|First Name|Last Name|Date|Holding|Value|
+    - The holding value can be calculated by `investments * amount`
```

Comment:
you called the field `investment` (singular) in the data. we should go with one or the other

Reply:
typo